packages/devtools_app/lib/src/flutter/status_line.dart

```diff
@@ -175,11 +175,28 @@ class StatusLine extends StatelessWidget {
                 '${vm.targetCPU}-${vm.architectureBits} ${vm.operatingSystem}';
           }
 
-          return Text(
-            'Connected ($description)',
-            style: textTheme.bodyText2,
-            overflow: TextOverflow.clip,
-            //maxLines: 1,
+          // TODO(devoncarew): Add an interactive dialog to the device status
+          // line.
+
+          return Row(
+            mainAxisAlignment: MainAxisAlignment.end,
+            children: <Widget>[
+              Text(
+                'Device: ',
+                style: textTheme.bodyText2,
+                overflow: TextOverflow.clip,
+              ),
+              Text(
+                description,
+                style: textTheme.bodyText2,
+                overflow: TextOverflow.clip,
+              ),
```

Comment:
nit: create a helper to eliminate the repeated code here and ensure style consistency

Reply:
I'll refactor this code a bit.